src/Supabase/AuthHelpers.purs

```diff
@@ -26,3 +32,28 @@ useUser = unsafeHook useUserImpl <#> mapRecordKind Nullable.toMaybe
 foreign import data UseUser :: Type -> Type
 
 foreign import createBrowserClient :: Effect Client
+
+type CookieOptions =
+  { domain :: Maybe String
+  , maxAge :: Maybe Int
+  , name :: Maybe Int
+  , path :: Maybe String
+  , sameSite :: Maybe String
+  , secure :: Maybe Boolean
+  }
+
+type Options =
+  { db :: Maybe String
+  }
+
+type ClientOptions =
+  { cookieOptions :: Maybe CookieOptions
+  , options :: Maybe Options
+  , supabaseKey :: Maybe String
+  , supabaseUrl :: Maybe String
+  }
+
+foreign import createBrowserClientWithOptionsInternal :: Foreign -> Effect Client
+
+createBrowserClientWithOptions ∷ ClientOptions → Effect Client
+createBrowserClientWithOptions = YogaJSON.write >>> createBrowserClientWithOptionsInternal
```

Comment:
How about 

```purescript
createBrowserClientWithOptions :: forall r r_. Union r r_ ClientOptions => {|r} -> Effect Client
createBrowserClientWithOptions = createBrowserClientWithOptionsImpl
```

And then `ClientOptions` is


```purescript
type ClientOptions = (
  cookieOptions :: CookieOptions
  , options :: Options
  , supabaseKey :: SupabaseKey
  , supabaseUrl :: SupabaseURL
  )
```


etc.?

Only in version 2?